docs/advanced/UsingRunSaga.md

```diff
@@ -7,20 +7,29 @@ When a Saga is started (either at startup or later dynamically), the middleware
 `redux-saga` provides a way to run a Saga outside of the Redux middleware environment and connect it to a custom Input/Output.
 
 ```javascript
-import { runSaga } from 'redux-saga'
+import { runSaga, stdChannel } from 'redux-saga'
 
-function* saga() { ... }
+const emitter = new EventEmitter();
+const channel = stdChannel();
+emitter.on("action", channel.put);
 
 const myIO = {
-  subscribe: ..., // this will be used to resolve take Effects
-  dispatch: ...,  // this will be used to resolve put Effects
-  getState: ...,  // this will be used to resolve select Effects
+  // this will be used to orchestrate take and put Effects
+  channel,
+  // this will be used to resolve put Effects
+  dispatch(output) {
+    emitter.emit("action", output);
+  },
+  // this will be used to resolve select Effects
+  getState() {
+    return state
+  }
 }
 
 runSaga(
   myIO,
-  saga,
+  function* saga() { ... },
 )
 ```
 
-For more info, see the [API docs](https://redux-saga.js.org/docs/api/index.html##runsagaoptions-saga-args).
+For more info, see the [API docs](https://redux-saga.js.org/docs/api/index.html##runsagaoptions-saga-args), [Channels](./Channels.md), [demo](https://codesandbox.io/s/1yq1lx77jq)
```

Comment:
codesandbox example is cool, we could make it an official example in the example directory and then we could point to it here with url like this:
https://codesandbox.io/s/github/redux-saga/redux-saga/tree/master/examples/run-saga-demo

Reply:
🚀 will experiment with it

Reply:
I's suggest to make in another PR.
I made it work with [changes](https://github.com/redux-saga/redux-saga/compare/update-docs...add-codesanbox-examples), just need to resolve some problems:
1 codesanbox doesn't install `devDependencies` https://github.com/CompuIves/codesandbox-client/issues/924 so I had to move all deps under `dependencies` field. 
2 `sagaMonitor` package is not published. we can switch to https://github.com/redux-saga/redux-saga-devtools or publish one from the example folder